packages/wxt/src/core/clean.ts

```diff
@@ -1,44 +1,48 @@
 import path from 'node:path';
 import glob from 'fast-glob';
 import fs from 'fs-extra';
-import { consola } from 'consola';
 import pc from 'picocolors';
+import { InlineConfig } from '~/types';
+import { registerWxt, wxt } from './wxt';
 
 /**
  * Remove generated/temp files from the directory.
  *
- * @param root The directory to look for generated/temp files in. Defaults to `process.cwd()`. Can be relative to `process.cwd()` or absolute.
+ * @param config Optional config that will override your `<root>/wxt.config.ts`.
  *
  * @example
  * await clean();
  */
-export async function clean(root = process.cwd()) {
-  consola.info('Cleaning Project');
+export async function clean(config?: InlineConfig) {
```

Comment:
This is a breaking change to the public API, but I agree it's probably better. Could you add an overload and mark the old implementation as deprecated?

I have one more breaking change to make for 0.19, I'll remove the deprecated overload when I'm preparing that release.